lib/web_ui/lib/src/engine/canvaskit/canvaskit_api.dart

```diff
@@ -1269,6 +1292,27 @@ SkFloat32List toMallocedSkPoints(List<ui.Offset> points) {
   return skPoints;
 }
 
+/// Converts a list of [ui.Offset] into a flat list of points.
+Float32List toFlatSkPoints(List<ui.Offset> points) {
```

Comment:
Do we still need `toSkPoints2d` and `rawPointsToSkPoints2d` after this PR? Also, can we use `toMallocedSkPoints` instead to avoid double-copying the data?

Reply:
I was unsure if that would mean we would have to `free` and re-`malloc` the positions whenever we delete the `SkVertices`. Since we only use it at once, at construction, I figured it wouldn't hurt performance to let CanvasKit do the `Malloc`.